### ISSUE TITLE
Uncaught SyntaxError: Unexpected token u in JSON at position 0

### ISSUE THREAD

**Issue by simpsons310** [taigaio/taiga-front#1766]
Hi, i'm new and trying to setup Taiga for development acorrding to this link: https://taigaio.github.io/taiga-doc/dist/setup-development.html
Everthing is done, but when i access http://localhost:9001, only a blank page show up. Then when i check error by press F12, it display error: 

> ticky-browser.js:8 Uncaught SyntaxError: Unexpected token u in JSON at position 0
    at JSON.parse (<anonymous>)
    at StorageService.get (storage.coffee:33)
    at HttpService.headers (http.coffee:33)
    at HttpService.request (http.coffee:45)
    at HttpService.get (http.coffee:56)
    at Object.Resource.service.getListProjectsByUserId (projects-resource.service.coffee:96)
    at ProjectsService.getListProjectsByUserId (projects.service.coffee:49)
    at CurrentUserService.loadProjectsList (current-user.service.coffee:74)
    at CurrentUserService._loadUserInfo (current-user.service.coffee:116)
    at CurrentUserService.setUser (current-user.service.coffee:63)
    at AuthService.setUserdata (auth.coffee:81)
    at new AuthService (auth.coffee:76)
    at Object.instantiate (ticky-browser.js:8)
    at Object.<anonymous> (ticky-browser.js:8)
    at Object.invoke (ticky-browser.js:8)
    at Object.enforcedReturnValue [as $get] (ticky-browser.js:8)

I have tried to delete taiga-front and reinstall but it couldn't solve the problem. I searched and found that it is a error with JSON but still stuck there. 
I'm using Version 74.0.3729.169 (Official Build) (64-bit)
Hope you solve this.

**Comment by alexhermida** [comment]
@simpsons310 review in detail your `conf.json` file (https://taigaio.github.io/taiga-doc/dist/setup-development.html#_final_steps) probably there is some issue. You could copy and modify the one from  https://github.com/taigaio/taiga-front/blob/master/conf/conf.example.json

**Comment by simpsons310** [comment]
> @simpsons310 review in detail your `conf.json` file (https://taigaio.github.io/taiga-doc/dist/setup-development.html#_final_steps) probably there is some issue. You could copy and modify the one from https://github.com/taigaio/taiga-front/blob/master/conf/conf.example.json

Thank for reply issue, actually, i solved this problem by upgrade my version of ubuntu from 16.04 to 18.04 then set up taiga again and it worked perfectly (my conf.json file is same with introduction document). Thus, I think it's due to Node Version Manager or Ruby Version Manger. Because, when I set up in Ubuntu 16.04, I had to install NVM and RVM to upgrade Ruby and NodeJS (It requires Ruby 2.4 at least to install sass and sccs_lint while default version is 2.3.x). Although I upgraded to Ruby 2.4.1 and Node 8.10.0, it still didn't work. So I upgraded my ubuntu and it worked.
